src/core/variables.jl

```diff
@@ -125,6 +125,9 @@ struct ComponentActivePowerReserveDownVariable <: SubComponentVariableType end
 # Necessary as a work around ofr HVDC models with losses
 struct HVDCTotalPowerDeliveredVariable <: VariableType end
 
+struct PieceWiseLinearCostVariable <: VariableType end
+
 const START_VARIABLES = (HotStartVariable, WarmStartVariable, ColdStartVariable)
 
 write_resulting_value(::Type{<:VariableType}) = true
+write_resulting_value(::Type{PieceWiseLinearCostVariable}) = false
```

Comment:
I don't recall where I commented before, but this function does not sound like a boolean.

Reply:
This is a comment in the base PR. Will be fixed there.